examples/ConfigurableFirmata/ConfigurableFirmata.ino

```diff
@@ -178,6 +178,9 @@ OneWireFirmata oneWire;
 #include <StepperFirmata.h>
 StepperFirmata stepper;
 
+//uncomment SERIAL_STORE_AND_FORWARD to collect bytes received by serial port until the receive buffer 
+// gets filled or a data gap is detected to avoid forwarding single bytes at baud rates below 50000
+//#define SERIAL_STORE_AND_FORWARD
```

Comment:
With Arduino, libraries are compiled before the sketch file runs so setting a define here does nothing. It needs to be set in the .h file.